### ISSUE TITLE
js picking suboptimal bounds for plot

### ISSUE THREAD

**Issue by bsoule** [beeminder/road#41]
ok, here's an example of one where the PY code does what I think is a more sensible job of picking bounds for the plot. (it looks like python is paying little attention to the road and mostly just paying attention to the data, though perhaps not entirely ignoring the road).

python:
<img width="325" alt="b_urlminder_drive_test-py" src="https://user-images.githubusercontent.com/46881/51002924-b77f0080-14e9-11e9-9e22-5434fcb1bbb9.png">

javascript:
<img width="335" alt="b_urlminder_drive_test-js" src="https://user-images.githubusercontent.com/46881/51002928-b948c400-14e9-11e9-9e41-c68e92ff5bdd.png">

bbfile:
```
{"params":{"tfin":"20200604","rfin":2.0,"road":[["20181112",null,0],["20181112",17,null],["20181115",null,500],["20181203",null,0],["20181203",107,null],["20181206",null,2],["20181227",null,0],["20181227",107,null],["20181230",null,2],["20190103",null,0],["20190103",107,null],["20190106",null,2],["20190109",null,0],["20190109",107,null]],"runits":"w","kyoom":false,"odom":true,"noisy":false,"yaw":1,"dir":1,"hashtags":false,"steppy":true,"rosy":false,"movingav":false,"aura":false,"stathead":false,"yaxis":"words","deadline":0,"aggday":"last","tagtime":false,"tmin":"20181203","plotall":true,"tini":"20181112","vini":90.0,"integery":false,"monotone":true,"hidey":false,"waterbux":"$5","yoog":"b/urlminder-drive-test","timezone":"America/Los_Angeles","imgsz":696,"asof":"20190110"},"data":[["20181112",90.0,"initial datapoint of 90.0 on the 12th"],["20181112",107.0,"Auto-entered via URLminder at 22:54"],["20181112",197.0,"Auto-entered via URLminder at 22:58"],["20181112",17.0,"Auto-entered via URLminder at 23:00"],["20181112",17.0,"RECOMMITTED ON THE 12th"],["20181112",107.0,"Auto-entered via URLminder at 23:01"],["20181112",17.0,"Auto-entered via URLminder at 23:04"],["20181112",107.0,"Auto-entered via URLminder at 23:12"],["20181203",107.0,"RESTARTED ON THE 3rd"],["20181227",107.0,"RECOMMITTED ON THE 27th"],["20190103",107.0,"RECOMMITTED ON THE 3rd"],["20190109",107.0,"RECOMMITTED ON THE 9th"]]}
```

**Comment by bsoule** [comment]
see another example of poorly chosen y-axis bounds in #42

**Comment by saranli** [comment]
Fixed with commit 7790f436351f869350e60237a102fb43aafd2cb3
